examples/playground/index.html

```diff
@@ -127,6 +143,7 @@ <h2 style="margin: 0; font-size: 1.2rem">Help</h2>
     </script>
 
     <a-scene
+      audio
```

Comment:
Actually you don't really need to add it if you don't set any option, and there is none. All systems are always included in the scene.